docs/api/index.md

```diff
@@ -1,2 +1,11 @@
-# PLACEHOLDER
-TODO: Add .NET projects to the *src* folder and run `docfx` to generate **REAL** *API Documentation*!
+# FoxGuard Foundation Libraries
+
+## About
+
+These are libraries that FoxGuard has developed over the course of several years of internal commercially-driven software engineering projects.
+
+For more information about the project, see [the project's GitHub]( https://github.com/foxguardsolutions/Foundation).
```

Comment:
there's a leading space in the hyperlink here that I don't think should be there

Reply:
✅